test/integration/local/events.test.js

```diff
@@ -1,3 +1,16 @@
+const mockSendMessage = jest.fn()
+
+jest.mock('ffc-messaging', () => {
+  return {
+    MessageSender: jest.fn().mockImplementation(() => {
+      return {
+        sendMessage: mockSendMessage,
+        closeConnection: jest.fn()
+      }
+    })
+  }
+})
+
 const { odata } = require('@azure/data-tables')
 const { PAYMENT_EVENT, HOLD_EVENT, BATCH_EVENT, WARNING_EVENT } = require('../../../app/constants/event-types')
 const { processEventMessage } = require('../../../app/messaging/process-event-message')
```

Comment:
I could only comment on these three lines, but could we put spaces between odata, constants and processEventMessage?